### PR TITLE
CONCF-669 Add temporary check to not break descriptions on production

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/CONCF-669

https://github.com/Wikia/mercury/pull/916 would break page descriptions on Mercury until https://github.com/Wikia/app/pull/7512 is pushed to production. This is a temporary solution to avoid errors.

/cc @bognix @tomasznapieralski

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by bognix** [comment]
LGTM

File: front/scripts/main/routes/ArticleRoute.ts [hunk 1]
```diff
@@ -36,7 +36,6 @@ App.ArticleRoute = Em.Route.extend({
 	model: function (params: any) {
 		return App.ArticleModel.find({
 			basePath: Mercury.wiki.basePath,
-			description: params.description,
```
Comment (rogatty) [inline front/scripts/main/routes/ArticleRoute.ts]: This isn't needed, description is being set in `ArticleModel#setArticle` method.

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**